REQUIRE

```diff
@@ -1,3 +1,3 @@
 julia 0.6
-Nulls
+Missings
```

Comment:
Does the package require Missings.jl with Julia >= 0.7? If not (as I suspect), better require `julia 0.7` and drop this dependency.

Reply:
Yes, I believe Missings.jl is only required for Julia 0.6. I was hoping there would be a version that works in 0.6 before moving to 0.7.0. (0.7 wasn't official yet)